test/bntest.c

```diff
@@ -154,6 +154,9 @@ int main(int argc, char *argv[])
     BIO *out;
     char *outfile = NULL;
 
+    CRYPTO_set_mem_debug(1);
```

Comment:
Same as the majority of test apps, no conditional on its activation. 
It's just test code.